instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -355,39 +434,55 @@ def handle_request(
         method, url, headers, stream, extensions = _extract_parameters(
             args, kwargs
         )
-        span_attributes = _prepare_attributes(method, url)
+        method_original = method.decode()
```

Comment:
What's the reason for the `decode()` addition?

Reply:
Actually we are already using `decode()` in L258 of the current implementation `_method = method.decode().upper()`. What I'm doing here is to get it as str to pass to the `_get_default_span_name` and later to the `sanitize_method`.